cirq/work/collector_test.py

```diff
@@ -52,7 +54,7 @@ def on_job_result(self, job, result):
     completion = TestCollector().collect_async(sampler=cirq.Simulator(),
                                                max_total_samples=100,
                                                concurrency=5)
-    cirq.testing.assert_asyncio_will_have_result(completion, None)
+    assert await completion is None
```

Comment:
Acknowledged, but the alternative interpretation results in a runtime error because bool is not awaitable. Rule of thumb is that `await` binds *super* tightly. Tighter than `x**y`, not as tight as `x[y]` or `x.y`.